src/main/java/com/spotinst/sdkjava/enums/aws/elastigroup/AwsMetricAlarmUnit.java

```diff
@@ -0,0 +1,67 @@
+package com.spotinst.sdkjava.enums.aws.elastigroup;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public enum AwsMetricAlarmUnit {
+
+    SECONDS("seconds"),
+    MICROSECONDS("microseconds"),
+    MILLISECONDS("milliseconds"),
+    BYTES("bytes"),
+    KILOBYTES("kilobytes"),
+    MEGABYTES("megabytes"),
+    GIGABYTES("gigabytes"),
+    TERABYTES("terabytes"),
+    BITS("bits"),
+    KILOBITS("kilobits"),
+    MEGABITS("megabits"),
+    GIGABITS("gigabits"),
+    TERABITS("terabits"),
+    PERCENT("percent"),
+    COUNT("count"),
+    BYTESORSECOND("bytes/second"),
```

Comment:
Can we change these to BYTESPERSECOND and similar for all following fields?

Reply:
resolved